consistent-streaming/README.md

```diff
@@ -0,0 +1,100 @@
+# YugabyteDB CDC Consistent Streaming Pipeline
+
+This repository contains simple steps to set up a YugabyteDB CDC consistent streaming pipeline to get the data from YSQL to a Kafka topic
+
+## Running the CDC pipeline
+The following example uses a PostgreSQL database as the sink database, which will be populated using JDBC Sink Connector.
+
+1. Start YugabyteDB
+    This can be a local instance as well as a universe running on Yugabyte Anywhere. All you need is the IP of the nodes where the tserver and master processes are running.
+    ```sh
+    export NODE=<IP-OF-YOUR-NODE>
+    export MASTERS=<MASTER-ADDRESSES>
+    ```
+  
+2. Create a table
+    This example uses two tables having a foreign key constraint. However, you can start with any tables of your choice.
+  
+    ```sql
+    CREATE TABLE department (id INT PRIMARY KEY, dept_name TEXT);
+    CREATE TABLE employee (id INT PRIMARY KEY, emp_name TEXT, d_id INT, FOREIGN KEY (d_id) REFERENCES department(id));
+    ```
+  
+3. Create a stream ID using yb-admin
+    ```
+    ./yb-admin --master_addresses $MASTERS create_change_data_stream ysql.<namespace> EXPLICIT
+    ```
+
+    **Note:** To use consistent streaming, you will need to create the stream in explicit checkpointing mode only.
+  
+4. Start the docker containers
+
+    ```sh
+    docker-compose up -d
+    ```
+  
+5. Deploy the source connector:
+
+    ```sh
+    ./deploy-sources.sh <stream-id-created-in-step-3> <comma-separated-list-of-tables>
+    ```
+
+    For example:
+    ```sh
+    ./deploy-sources.sh 65a857d957f34507aa4fd97ebcb6ff56 public.department,public.employee
+    ```
+
+    #### Key considerations
+    The following configuration properties are of key importance while deploying the source connector:
+
+    | Config property | Default | Description |
+    | :--- | :--- | :--- |
+    | `provide.transaction.metadata` | `false` | Send metadata related to transaction boundaries |
+    | `transaction.ordering` | `false` | Whether to sort transactions in connector |
+  
+6. Deploy the sink connector
+    
+    ```sh
+    ./deploy-sinks.sh $NODE
+    ```
+
+    #### Key considerations
```

Comment:
Provide link to repo and also small paragraph on changes done. Highlight that this just for testing purpose.